### DISCUSSION TITLE
Equipment tracking

### DISCUSSION THREAD

**Discussion by crcrewso** [qatrackplus/qatrackplus#558] [Ideas]
The SCC is looking at modernizing our equipment inventory. This would include:

- Product, model, and serial number
- Manufacturer and Supplier
- Manufacture date
- Commission/acceptance date
- end of warrantee date
- If it requires external calibration or verification
- Maybe calibration factors, though this is a stretch

I was wondering if there is interest in building it into QA track, partly so

- Service events could be created for sending equipment away for repair or maintenance. 
- Active vs retired status of equipment, ie we got new thing.2 but thing.1 still works, where should we store it, is it still useful. 

Right now all of this information is stored either in spreadsheets or in the collective memory. 

Do people like this idea? Does it make sense? What other functionality would make sense? Is this not worth the effort?

**Comment by randlet** [comment]
This is a pretty highly requested feature.   There is a not flushed out issue here: https://github.com/qatrackplus/qatrackplus/issues/528
